log/src/test/java/kcl/teamIndexZero/traffic/log/fileIO/FileInputTest.java

```diff
@@ -24,7 +25,7 @@ public void tearDown() throws Exception {
 
     @Test
     public void testRead() throws Exception {
-        FileOutput out = new FileOutput("", "testRead.test");
+        kcl.teamIndexZero.traffic.log.fileIO.FileOutput out = new kcl.teamIndexZero.traffic.log.fileIO.FileOutput("", "testRead.test");
```

Comment:
Ok, it's just "and that's not needed too" so can't really get a hold of what you are trying to say. However I never changed anything in the log files, I only changed things in the guiprimitives, simulationimageproducer, and the guiprimitivetest, nothing else. So don't know what is the problem in log, but let me see the complete history on web and get back to you on that one.                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: Alexander CherednichenkoSent: Monday 21 March 2016 08:11To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In log/src/test/java/kcl/teamIndexZero/traffic/log/fileIO/FileInputTest.java:
> @@ -24,7 +25,7 @@ public void tearDown() throws Exception {
>  
>      @Test
>      public void testRead() throws Exception {
> -        FileOutput out = new FileOutput("", "testRead.test");
> +        kcl.teamIndexZero.traffic.log.fileIO.FileOutput out = new kcl.teamIndexZero.traffic.log.fileIO.FileOutput("", "testRead.test");

and that's not needed too

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub